x-pack/plugins/cases/server/client/cases/find.ts

```diff
@@ -85,11 +87,18 @@ export const find = async (
 
     const caseQueryOptions = constructQueryOptions({ ...queryArgs, authorizationFilter });
 
+    const caseSearchById = constructSearchById(
+      queryParams.search ?? '',
```

Comment:
nit: You can pass `undefined` instead of an empty string and `constructSearchById` can return `undefined` if that is the case. If you spread `undefined` into an object it will not be added by JS (I know crazy).